stake-house/wagyu-key-gen
# Updating yarn to v4.1.0

Upgrading to yarn v4.1.0 and adding `enableHardenedMode: true` to `.yarnrc.yml` to protect against lockfile dependency 
injection attacks: https://github.com/yarnpkg/berry/discussions/4136

To test:
Pull refresh version of this branch
Make sure running >= Node v18
`yarn install`
`yarn buildcli`
`yarn build`
`yarn start`

Should run with no regressions and no local files should be modified
